libmscore/text.cpp

```diff
@@ -2659,7 +2659,12 @@ Element* Text::drop(const DropData& data)
                         }
                   else {
                         startEdit(data.view, data.pos);
-                        curLine().insert(_cursor, QChar(code));
+                        if (QChar::requiresSurrogates(code)) {
```

Comment:
I also should say I am unable to how to exercise this else block.  I guessing no elements will accept this drop?